src/client/assets/javascripts/app/routes.js

```diff
@@ -5,11 +5,13 @@ import App from './App';
 import ActionEnter from 'components/ActionEnter';
 import AddAction from 'components/AddAction';
 import NotFoundView from 'components/NotFound';
+import Welcome from 'components/Welcome';
 
 export default (
 	<Route path="/" component={App}>
-		<IndexRoute component={AddAction} />
+		<IndexRoute component={Welcome} />
 		<Route path="404" component={NotFoundView} />
+		<Route path="select" component={AddAction} />
```

Comment:
@rochellecurrier this is why it's skipping the type selection section. `/select` is a route, and then `/:focus` is a route, and then `:focus/:type` is a route, they are all siblings at the same level.

`:focus` and `:focus/:type` need to be children of `select` because they apply underneath the `/select` section. Otherwise, after you pick a focus it will match the `:focus/:type` pattern in this format: `/select/Immigration`